src/models/Permissions.js

```diff
@@ -0,0 +1,47 @@
+import {Model} from 'sequelize';
+
+module.exports = (sequelize, DataTypes) => {
+  // eslint-disable-next-line require-jsdoc
+  class Permissions extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+     static associate({Roles}) {
+      this.belongsTo(Roles, { foreignKey: "roleId"});
+      // define association here
+    }
```

Comment:
I think we have many_to_many relationships on creating roles and permissions, though you have used one_to_many. check well the Trello board here https://trello.com/c/zRlfxZP8/9-create-roles-and-set-permissions